sandbox.sh

```diff
@@ -3,15 +3,12 @@
 
 set -e
 
-if [ "$2" = "docker" ]
-then
+if [ "$2" = "docker" ]; then
```

Comment:
Any idea of the purpose of this?
I didn't even know it is possible to do 
```shell script
$ ./sandbox.sh setup docker
# or
$ ./sandbox.sh start local
```

Reply:
We want to be able to run a Deku cluster from docker-compose. The settings are different, namely the URL's. This is a quick hack to make the script work both locally and in docker compose.